HOW_WE_WORK.md

```diff
@@ -0,0 +1,51 @@
+# HOW_WE_WORK
+
+We believe in transparent, collaborative development. These guidelines document how we work together and with our community.
+
+## Workflow
+
+### Issues & Discussions
+
+- Use **Issues** for bugs and features, **Discussions** for RFCs and support
+- Only Issues appear on our Project Board
+- Cross-reference PRs, issues, and discussions to maintain context
+
+### Pull Requests
+
+- Focus each PR on a single concern
+- Use Draft PRs for work-in-progress changes
+- Clearly flag breaking changes and update relevant documentation
+- Notify the team when PRs are ready for review
+- Request reviews from specific maintainers based on expertise
+
+### Releases
+
+- Schedule releases Monday-Wednesday only
```

Comment:
@josephfusco 

Sorry I know we agreed Monday to Wednesday, but I wanted to suggest Thursday aswell.

Reply:
I'm not against this, might be beneficial to define a timezone to guard against a Thursday evening release. Thoughts?

Reply:
@josephfusco 

That is a good point. Maybe anytime before the team meeting at 4pm GMT?

Also happy to leave it as is.